docs/mkdocs.yml

```
@@ -32,6 +32,7 @@ nav:
       - Media Backend:
           - MinIO: guides/minio-image-upload.md
           - S3: guides/s3-image-upload.md
+          - Google Cloud Storage Buckets: guides/s3-google-cloud-storage.md
```

That's shorter and precise enough

```suggestion
          - Google Cloud Storage: guides/s3-google-cloud-storage.md
```